Product/Production/Services/AdminDistributionCore/src/main/java/gov/hhs/fha/nhinc/admindistribution/inbound/AbstractInboundAdminDistribution.java

```diff
@@ -44,7 +44,7 @@
     protected AdminDistributionUtils adminUtils = AdminDistributionUtils.getInstance();
     protected AdapterAdminDistributionProxyObjectFactory adapterFactory = new AdapterAdminDistributionProxyObjectFactory();
 
-    abstract void processAdminDistribution(EDXLDistribution body, AssertionType assertion);
+    public abstract void processAdminDistribution(EDXLDistribution body, AssertionType assertion);
```

Comment:
We encountered several issues with similar msg like this "org.springframework.aop.framework.CglibAopProxy] (MSC service thread 1-6) Unable to proxy method[...] because it is package-visible across different ClassLoaders: All calls to this method via a proxy will NOT be routed to the target instance.".   The spring framework(4.1.9) introduce new feature in CglibAopProxy to detect package visible methods when defined in a different classloader.  It displays warning message if method is not final/protected/private and loaded in different classloader.  We resolved this msg by defining public method in those aspect classes.  https://github.com/spring-projects/spring-framework/blob/3370f41c610a0b2f600a2112a6a9ddaf6cf93796/spring-aop/src/main/java/org/springframework/aop/framework/CglibAopProxy.java